### PR TITLE
refactor!: removed all deprecated members

### PR DESCRIPTION
**Pull request by daadu**

### CODE REVIEW

File: lib/scaffold.dart [hunk 1]
```diff
@@ -669,16 +562,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
...
-              title: widget.title,
-              actions: widget.iconPosition == BackdropIconPosition.action
-                  ? <Widget>[const BackdropToggleButton()] + widget.actions
-                  : widget.actions,
-              elevation: 0,
-              leading: widget.iconPosition == BackdropIconPosition.leading
-                  ? const BackdropToggleButton()
-                  : null,
-            ),
+            AppBar(elevation: 0, leading: const BackdropToggleButton()),
```
Comment (daadu) [inline lib/scaffold.dart:565]: @WieFel This will enforce having an `appBar` - should we remove the default one?

Comment (WieFel) [reply lib/scaffold.dart:566]: good point actually. I would say yes, because backdrop should also be usable without an app bar

Comment (daadu) [reply lib/scaffold.dart:566]: Ok.

**Comment by daadu** [comment]
@WieFel  All changes are done. Please review it.

**Review by WieFel — APPROVED** [review]

**Pull request merged**